src/CSFML/Audio/CMakeLists.txt

```
@@ -4,6 +4,8 @@ set(SRCROOT ${PROJECT_SOURCE_DIR}/src/CSFML/Audio)
 # all source files
 set(SRC
     ${INCROOT}/Export.h
+    ${INCROOT}/EffectProcessor.h
+    ${SRCROOT}/ConvertCone.hpp
```

```suggestion
    ${SRCROOT}/ConvertCone.hpp
    ${INCROOT}/EffectProcessor.h
```
Let's keep the source files in alphabetical order.